platform/WORKSPACE

```diff
@@ -115,11 +115,15 @@ go_repository(
 
 go_repository_alt(
     name = "com_github_coreos_etcd",
-    commit = "06cec40911744c0aca5d1afd69b2631b0a4879ba", # 3.2.26
+    commit = "d57e8b8d97adfc4a6c224fe116714bf1a1f3beb9", # 3.3.12
     importpath = "github.com/coreos/etcd",
     build_external = "vendored",
     build_file_proto_mode = "disable_global",
-    prepatch_cmds = ["mv -Tn cmd/vendor vendor"],  # to get etcd's vendoring strategy to be compatible with gazelle
+    prepatch_cmds = [ # to get etcd's vendoring strategy to be compatible with gazelle
+        "mv -T vendor/github.com/grpc-ecosystem/go-grpc-middleware cmd/vendor/github.com/grpc-ecosystem/go-grpc-middleware",
+        "rmdir vendor/github.com/grpc-ecosystem && rmdir vendor/github.com && rmdir vendor",
+        "mv -T cmd/vendor vendor"
+    ],
```

Comment:
pretty sure there's a better way to do this, like `cp -rT cmd/vendor vendor; rm -r cmd/vendor ` or something

Reply:
Will give it a try and update the PR if it works.

Reply:
It works.